docs/source/api_ref.rst

```diff
@@ -0,0 +1,8 @@
+IDE Parser API Reference
```

Comment:
Personally, I'd just say `idelib`, since it does more than parse.

Reply:
do you have a preference of "idelib" vs. "IDELib" vs. something else? normally I'd say the former, but even numpy is more formally "NumPy" outside of code imports.

Reply:
I think it makes sense to call it `idelib` (shown in 'code style'). It's basically arbitrary, but I've seen various other packages reference themselves by their actual import name. NumPy is almost always imported under a different name, which might be factor in how it's referenced in its docs.

Reply:
`idelib` it is. changed in e2e04b7f355b13db458c3cf8104d6a3304f8d32c